achievements/migrations/0002_auto_20190826_2224.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 2.2.3 on 2019-08-26 20:24
```

Comment:
So many migrations, isn't there a better way?

Reply:
You can squash the migrations (https://docs.djangoproject.com/en/2.2/topics/migrations/#migration-squashing). Might be good to keep the initial migration (the one regarding the membership file) as a separate migration though.

Reply:
It might be an idea to just not worry about it now and when we are ready to release the first version of the app we just squash them? This way we will not lose a lot of time on squashing and re-squashing again and again?

Reply:
If you look at your CI build out, you can see that it is related to database migrations. Therefore I suggest you fix this issue while you are fixing the build.

Also, ever heard of the Broken Window theory?

Reply:
No and I also do not know what I would have to do to 'fix the build'

Reply:
Looking at the build output, which you can find in the check menu, you can see that the build crashes with `django.db.utils.OperationalError: no such table: achievements_test`. No clue what happened there.

Reply:
I will have a look then